translations/wesnoth-A_New_Order/es.po

```diff
@@ -75,18 +81,22 @@ msgstr "Oráculo"
 #: A_New_Order/units/others/Oracle.cfg:21
 msgid "No description. It will never appear anywhere, since the unit is fake."
 msgstr ""
+"Sin descripción. Nunca aparecerá en ninguna parte, ya que la unidad es falsa."
```

Comment:
Hm, I guess this one doesn't actually need to be translatable/translated, if it's as it says... I'll add a translation note to that effect... (Since you already translated that, though, that's fine, it can stay)

Reply:
done in my other repo